Jenkinsfile

```diff
@@ -1,38 +1,12 @@
-node('docker') {
-
-    stage("Git checkout"){
-        checkout scm
-    }
-
-    stage('Cleaning') {
-        try {
-            sh "docker rm -f ibaa-chain ibaa-broker ibaa-blockchain-adapter-mongo"
-        } catch (err) {
-            echo err.getMessage()
-        }
-        try {
-            sh "docker network create iexec-blockchain-net"
-        } catch (err) {
-            echo err.getMessage()
-        }
-    }
-
-    stage('Setup') {
-        withCredentials([string(credentialsId: 'BROKER_PRIVATE_KEY', variable: 'brokerPrivateKey')]) {
-            sh "BROKER_PRIVATE_KEY=${brokerPrivateKey} docker-compose up -d"
-        }
-    }
-
-    stage('Test') {
-        sh './gradlew build itest -i --no-daemon'
-        junit 'build/test-results/**/*.xml'
-    }
-
-    stage('Teardown') {
-        sh 'docker-compose down'
-    }
-
-}
-
-@Library('jenkins-library@master') _
-buildSimpleDocker(imageprivacy: 'public')
\ No newline at end of file
+@Library('jenkins-library@feature/build-java-project') _
+buildJavaProject(
+        shouldRunIntegrationTests: true,
+        integrationTestsEnvVars: ["BROKER_PRIVATE_KEY"],
+        shouldPublishJars: true,
```

Comment:
Do we publish jars for all projects?

Reply:
I think we don't need it in this particular commit, but we might need it for publishing POJO classes of exposed endpoints (see current PRs consuming iexec-common instead of blockchain-adapter-lib)

Reply:
Disabled JARs upload for now

Reply:
Thanks!